lib/controllers/coinstac/consortia.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const config = require('config'); // jshint ignore:line
+const _ = require('lodash');
+const consortiaDbOptions = config.get('coinstac.pouchdb.consortia');
+
+const internals = {};
+
+internals.consortiaSchema = joi.object().keys({
+    label: joi.string().required(),
+    users: joi.array().required(),
+    description: joi.string().required(),
+    tags: joi.array().required(),
+    analyses: joi.array().required()
+});
+
+internals.savedConsortiaSchema = internals.consortiaSchema.keys({
+    _id: joi.string().required(),
+    _rev: joi.string().required()
+});
+
+internals.createConsortiaDb = (name, PouchW) => {
+    const defaults = consortiaDbOptions;
+    let pouchWConfig;
+
+    //TODO: return name of db for ease of location
+    name = defaults.name + '-' + _.kebabCase(name.toLowerCase());
+    if (consortiaDbOptions.path) {
+        pouchWConfig = {
+            path: defaults.path,
+            name: name
+        };
+    } else if (consortiaDbOptions.conn) {
+        pouchWConfig = {
+            name: name,
+            conn: defaults.conn
+        };
+    }
+
+    // create new db
+    const newDb = new PouchW(pouchWConfig);
+    return newDb;
+};
+
+module.exports.getAll = {
+    tags: ['api', 'coinstac', 'consortia'],
+    notes: [
+        'Returns all consortia stored in consortia DB',
+        'No query params supported currently'
+    ].join('<br>'),
+    description: 'Returns array of all consortia',
+    validate: {
+        query: joi.object().keys({})
```

Comment:
Leaving it empty will enforce that no query params are sent with the request. This does two things:

1. The swagger docs will explicitly show that query params are not supported.
1. A request with query params will fail, thus alerting the client-side developer that we don't use their query params. If we just disregard the query params, they may not realized that `&limit=10` is not respected by this route.